### PR TITLE
Update Sign Up Page

### PR DESCRIPTION
**Pull request by victoriaklo**
## Ticket Description
<-- Copy and paste or write the tickets description here-->
PR for #141 
Signup page has minor cleanup that needs to be done:

- [x]  mobile signup on Figma not present, but if needed, change: (see notes tab in excel spreadsheet underneath saige, signup page)
- [x]  need to add Data Umbrella copyright tag on footer
- [x]  change Data Umbrella logo to colored in dark mode
- [x]  mobile signup on Figma not present, but if needed, change: (see notes tab)
- [x]  terms & conditions hyperlink leads to empty page

## Description of Changes
<-- Brief description of the changes you made, files you touched, etc -->
## Before and After for UI Updates
<-- You should have 2 images minimum, for desktop, 4 for mobile and desktop combined -->

Before:
Screen Shot 2022-09-12 at 10 48 00 AM

After:
Screen Shot 2022-09-12 at 11 12 49 AM

## [107974e](https://github.com/data-umbrella/event-board-web/pull/168/commits/107974eadc28e34c8a4625ef83ea710c3f5e0a8c)

Before:
Add link to terms page
Screen Shot 2022-09-12 at 11 27 34 AM

After:
Screen Shot 2022-09-12 at 11 27 02 AM

## [160f815](https://github.com/data-umbrella/event-board-web/pull/168/commits/160f81507679ab424ea7f17a95423c7b48fc4611)
Small fix for mobile view
Before:
Screen Shot 2022-09-12 at 4 30 49 PM

After:
Screen Shot 2022-09-12 at 4 30 32 PM

## For PR Reviewer
- [ ] Does this file change the yarn.lock, package.json or package-lock.json file? If so, why?
- [ ] If this pr contains mobile and desktop changes, did you test on IphoneXr and desktop views?
- [ ] Does this file match the related tickets linked figma file, or does it pass the visual smell test?
- [ ] If this file contains javascript, does the javascript pass the smell test?
- [ ] If you don't feel super confident in your review, did you assign someone more senior to double check?

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -13,6 +13,9 @@ module.exports = {
           600: '#783D86',
         },
         'du-gray': '#F7FAFC',
+        'indigo':{
```
Comment (dashcraft) [inline tailwind.config.js:16]: use the `du-` prefix for indigo so we don't override default tailwind styles if one exists

File: src/components/elements/Footer.jsx [hunk 2]
```diff
@@ -33,10 +33,11 @@ function Footer() {
           <ul className="grid lg:grid-rows-3 lg:gap-4">
             <li className="mb-2"><a href="https://www.dataumbrella.org">Data Umbrella</a></li>
             <li className="mb-2"><a href="/privacy">Privacy</a></li>
-            <li className="mb-2">Terms & Conditions</li>
+            <li className="mb-2"><a href="/terms">Terms & Conditions</a></li>
           </ul>
         </div>
       </div>
+      <div className="text-white text-center pb-5">	<p>&copy; Data Umbrella 2022</p></div>
```
Comment (dashcraft) [inline src/components/elements/Footer.jsx:40]: comment (non blocking):
remove white space

File: src/components/elements/Footer.jsx [hunk 3]
```diff
@@ -33,10 +33,11 @@ function Footer() {
           <ul className="grid lg:grid-rows-3 lg:gap-4">
             <li className="mb-2"><a href="https://www.dataumbrella.org">Data Umbrella</a></li>
             <li className="mb-2"><a href="/privacy">Privacy</a></li>
-            <li className="mb-2">Terms & Conditions</li>
+            <li className="mb-2"><a href="/terms">Terms & Conditions</a></li>
```
Comment (dashcraft) [inline src/components/elements/Footer.jsx:36]: Comment (blocking):
Go ahead and change all of these links right here specifically, to use the react router Link element
Only use anchor links for external pages when writing react (unless its nextjs)

**Review by dashcraft — CHANGES_REQUESTED** [review]
Some blocking and non blocking comments

There's also a compiler error, might make sure this is ok:

src/components/elements/AppRoutes.jsx
  Line 70:38:  'TermsPage' is not defined  react/jsx-no-undef

**Comment by victoriaklo** [comment]
@dashcraft 
RE: src/components/elements/AppRoutes.jsx
Line 70:38: 'TermsPage' is not defined react/jsx-no-undef

I commented this part out and this will be added in the terms PR #171 instead

**Review by dashcraft — APPROVED** [review]
Rebase this pr before merging

**Review by dashcraft — APPROVED** [review]

**Pull request merged**